revised/stellaswap-proposal

```diff
@@ -0,0 +1,237 @@
+* Worked with DTMB & Moonwell to establish Moonbeam’s DeFi Voyage NFT campaign to spur activity within the ecosystem
+* Worked with parachains such as Acala, Phala, Astar to launch their native tokens on Moonbeam
+
+__CONNECTED CONTRACTS USE CASE__ (Updated Feedback from Turritz)
+This is something we’ve been EXTREMELY EXCITED about! For context, we’ve been working with Squid since last quarter of 2022 to lay the infrastructure + provide optimizations for crosschain swaps (CCS). Kudos to the Squid team for their thought-leadership and responsiveness in ensuring CCS was perfect. The coordinated launch of CCS swaps on Moonbeam was a huge landmark for the ecosystem, and StellaSwap was the first to launch CCS powered by Squid & Axelar. The importance of CCS in abstracting complexities and simplifying DeFi was the main reason why we were extremely enthusiastic about CCS and pushed forward a slew of targeted marketing campaign (e.g. with Moonbeam, Polygon, Arbitrum) to drive adoption and capital inflow towards Moonbeam. We’re proud to say that CCS on Moonbeam has been a great success, with Moonbeam leading the charts on usage and capital inflows:
+
+![image](upload://oOQ764w6rEzOrGDbYk147ohuYpD)
+
+We will continue to induce behaviour towards using CCS, especially for first-time users and draw users toward Moonbeam, with the help of Squid+Axelar+participating ecosystems.
+
+Beyond CCS as the base layer utility of connected contracts that just went live on Moonbeam on StellaSwap, we’re excited to also explore greater usecases that is built on Squid+Axelar’s infrastructure, such as one-click ZAP into LPs or borrowing protocols. With protocols like Prime going live very soon focused on cross-chain borrowing/lending, we’re excited to establish the liquidity infrastructure to power their use cases.
+ 
+__GRANTS PERFORMANCE__ 
+Let's take a look at the performance of the previous grants cycle.
+
+**TVL Targets** (Updated feedback from Natalia)
+**![image|690x290](upload://7JZHLEO8JdW9biilsRr2c8avWjb.jpeg)**
+Our initial forecasted target for the first 4 weeks was $26M, which we promptly hit as we topped at around $28M. From then on, the market turned north as GLMR tanked approximately 40% in price from a high of $0.53 to a low of $0.32 in a span of two weeks. General market conditions were bleak, with the entire market bleeding along the same ranges. This impacted our TVL growth, as TVL is fuelled by 1) the USD price of GLMR & STELLA emissions as well as 2) USD price of all assets on our farm. 
+
+**![|267x291](upload://5zpUjxqAfj7z2u0aFKuI4PagxCR.png)**
+Market drawdown was aligned to the reduction of our TVL growth forecasts.
+
+**Total Volume**
+**![|602x273](upload://zmV7IuXpLCmvbHyOY0OyFecCfHw.png)**
```

Comment:
Can you add the image to the PR - otherwise this will show a blank image placeholder.